lib/elastic_searcher.rb

```diff
@@ -14,6 +14,15 @@ def search
     @api ? result : [error_msg(e), result]
   end
 
+  def suggest
```

Comment:
Please check for error (search[0]) here, and add unit tests for this.
Also, perhaps a better name could be `suggestions`?